android/gradle/config.gradle

```diff
@@ -6,13 +6,16 @@ ext {
     targetSdkVersion = 30
     compileSdkVersion = 30
     buildToolsVersion = "30.0.3"
+    quickAuthMetricLibVersion = "v0.1"
```

Comment:
Is the `v` necessary? Can we just use `0.1`?

Reply:
Not necessary, but I think we can keep the same naming format as the web with the vxxx